### PR TITLE
Todo-16-BE: As a user, I want to edit todos

### PR DESCRIPTION
**Pull request by itc02**

### CODE REVIEW

File: app/services/update_todo_service.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class UpdateTodoService < ActiveInteraction::Base
   string :description 
   integer :assigned_to
   date :deadline
-  state :string
+  string :state
```
Comment (arpinehp) [inline app/services/update_todo_service.rb:7]: :smile:

File: app/controllers/todos_controller.rb [hunk 2]
```diff
@@ -39,18 +41,18 @@ def destroy
 
   def update
     result = UpdateTodoService.run(
-      :id => params[:id],
+      :id => params[:id].to_i,
       :title => params[:title], 
       :description => params[:description], 
-      :assigned_to => params[:assigned_to], 
+      :assigned_to => params[:assigned_to].to_i,
```
Comment (arpinehp) [inline app/controllers/todos_controller.rb]: maybe `user_id`?

File: app/controllers/todos_controller.rb [hunk 3]
```diff
@@ -8,10 +8,12 @@ def index
     )
 
     if todos.valid?
-      render json: todos.result
+      render :json => todos.result
```
Comment (arpinehp) [inline app/controllers/todos_controller.rb:11]: what the differences and why have you changed it?

**Review by arpinehp — CHANGES_REQUESTED** [review]

File: app/controllers/todos_controller.rb (hunk 3 above)
Comment (itc02) [reply app/controllers/todos_controller.rb:11]: There is no difference. I just like to write in such manner :)

Comment (arpinehp) [reply app/controllers/todos_controller.rb:11]: okay

**Review by arpinehp — APPROVED** [review]

**Pull request merged**